Makefile.in

```diff
@@ -864,6 +880,42 @@ stamps/build-qemu: $(QEMU_SRCDIR) $(QEMU_SRC_GIT)
 	mkdir -p $(dir $@)
 	date > $@
 
+stamps/build-llvm-linux: $(LLVM_SRCDIR) $(LLVM_SRC_GIT) \
+                         stamps/build-gcc-linux-stage2
+	# We have the following situation:
+	# - sysroot directory: $(INSTALL_DIR)/sysroot
+	# - GCC install directory: $(INSTALL_DIR)
+	# However, LLVM does not allow to set a GCC install prefix
+	# (-DGCC_INSTALL_PREFIX) if a sysroot (-DDEFAULT_SYSROOT) is set
+	# (the GCC install prefix will be ignored silently).
+	# Without a proper sysroot path feature.h won't be found by clang.
+	# Without a proper GCC install directory libgcc won't be found.
+	# As a workaround we have to merge both paths:
+	mkdir -p $(SYSROOT)/lib/
+	cp -a $(INSTALL_DIR)/lib/gcc $(SYSROOT)/lib/
+	rm -rf $@ $(notdir $@)
+	mkdir $(notdir $@)
+	cd $(notdir $@) && \
+	    cmake $(LLVM_SRCDIR)/llvm \
+	    -DCMAKE_INSTALL_PREFIX=$(INSTALL_DIR) \
+	    -DCMAKE_BUILD_TYPE=Release \
+	    -DLLVM_TARGETS_TO_BUILD="RISCV" \
+	    -DLLVM_ENABLE_PROJECTS="clang;lld" \
+	    -DLLVM_ENABLE_RUNTIMES="libcxx;libcxxabi;libunwind" \
+	    -DLLVM_DEFAULT_TARGET_TRIPLE="$(LINUX_TUPLE)" \
+	    -DDEFAULT_SYSROOT="$(INSTALL_DIR)/sysroot" \
```

Comment:
This won't work, because with "../sysroot" the sysroot is not found during LLVM's build process.

Reply:
Oh, damm...one possible solution is we create a symbolic link in the build folder to resolve this issue? otherwise the toolchain will broken when move to other place other than `$(INSTALL_DIR)`

Reply:
I always take 2 stage build, first for clang/llvm and then build those run-time library in second stage, so we didn't got such issue before...

Reply:
it could be `../sysroot` after adding symbol link in build folder, checked on my build machine.